### PR TITLE
Add recipe for snakeviz

### PR DESCRIPTION
**Pull request by gforsyth**

### CODE REVIEW

**Comment by gforsyth** [comment]
@jiffyclub let me know if you want to be a maintainer of this recipe -- wanted to check before submitting you to the conda-forge emails

**Comment by gforsyth** [comment]
@conda-forge/help-python CI checks have passed and this recipe is ready for review.  Thanks!

File: recipes/snakeviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "snakeviz" %}
+{% set version = "v2.0.0" %}
```
Comment (beckermr) [inline recipes/snakeviz/meta.yaml]: Let's remove the `v` here.

Comment (gforsyth) [reply recipes/snakeviz/meta.yaml]: Happy to, though that is the tagged release "number" -- so there'll be a `v{{ version }}` in the source url

Comment (beckermr) [reply recipes/snakeviz/meta.yaml]: Right yeah the URL is fine. We just want the `v` not in the official version.

**Comment by gforsyth** [comment]
Thanks @beckermr !

**Comment by jiffyclub** [comment]
Thanks for the ping @gforsyth! I'm happy to let you handle this, but let me know if there's anything I can help with.

**Pull request merged**